packages/mds-ingest-service/repository/entities/event-annotation-entity.ts

```diff
@@ -47,6 +48,13 @@ export class EventAnnotationEntity extends IdentityColumn(RecordedColumn(class {
 
   @Column('bigint', { transformer: BigintTransformer })
   latency_ms: Timestamp
+
+  @OneToOne(type => EventEntity, event => event.annotation, { nullable: true })
+  @JoinColumn({ name: 'events_row_id', referencedColumnName: 'id' })
+  event?: EventEntityModel
+
+  @Column('bigint')
+  events_row_id?: number
```

Comment:
And this should always exist, right?

Reply:
Also, `bigint` columns should have the `{ transformer: BigintTransformer }` option

Reply:
Added transformer.